templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
+[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
+
+set -eu -o pipefail
+
+ocpath='{{ owncloud_home }}'
```

Comment:
Good idea :+1:. I guess that should be something like this to make it work through:

```shell
ocpath='{{ owncloud_home }}'
ocpath="${ocpath:-/var/www/owncloud}"
```